spec/test_files/get_commits_since_last_release_patch_changes.json

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
I added some new responses because I wanted to simulate a patch release, so I got a bunch of commits that represented a patch. I didn't want to make it up so I got a real response